### DISCUSSION TITLE
Get body and screenshots for all redirections

### DISCUSSION THREAD

**Discussion by MrWazaby** [go-rod/rod#1140] [Q&A]
Hello everyone, 

I have a website doing that:
`http://site1/page -> (http to https redirect) -> https://site1/page -> (redirect via headers) -> https://site2 -> (redirect via JS) -> https://site2/page`

At the end, by putting `http://site1/page` I've go the screenshot and body of `https://site2/page` with `MustScreenshot() `and `MustHTML()`. I also retrieve the response headers. 

Do you know a way so I can get html body, headers and screenshots (if any) of each steps?

**Comment by 0x123456789** [comment]
@MrWazaby Hi! Did you find a solution?
